qldbdriver/qldbdriver.go

```diff
@@ -201,13 +201,6 @@ func (driver *QLDBDriver) getSession(ctx context.Context) (*session, error) {
 	driver.logger.log(fmt.Sprint("Getting session. Existing sessions available:", len(driver.sessionPool)), LogDebug)
 	isPermitAcquired := driver.semaphore.TryAcquire()
 	if isPermitAcquired {
-		defer func(driver *QLDBDriver) {
```

Comment:
You can't remove this.

If the customer decides to panic in their lambda, without this the entire driver would end up being broken.

Reply:
How is it possible for a lambda function panic to happen here given that there is no such lambda in scope in `func (driver *QLDBDriver) getSession(ctx context.Context) (*session, error)`?